README.md

```diff
@@ -127,6 +127,40 @@ It integrates naturally with Python and Ruby test frameworks.
 It is fast! It will not increase the time of your tests beyond the acceptable.
 
 
+## How to build
+
+These instruction are currently tested on Ubuntu 14.04. Other distros might work just fine, but they
+were never tested, so 14.04 should give you the most trustworthy results.
+
```

Comment:
It is important to mention that it is trusted to work with Python 2.7. The code may need adjusts to run with Python 3, and Python 2.6 may work, but it is not tested.